components/brave_vpn/brave_vpn.mojom

```diff
@@ -42,6 +42,7 @@ interface ServiceHandler {
   Connect();
   Disconnect();
   GetAllRegions() => (array<Region> regions);
+  GetDefaultRegionName() => (string default_region_name);
```

Comment:
This should be returning the Region struct and perhaps rename the function to GetDefaultRegion

Reply:
How about making `struct Region` has `bool is_default` instead of adding addtional method?
Below is better for webui?
```
struct Region {
  string continent;
  string name;
  string name_pretty;
  bool is_default;
};
```

Reply:
maybe `is_default` can be mistaken as user's previous selection. for ex, if you select "japan" as region and close the browser should your profile remember that as default now? thinking about future case here.

I think what we're doing here getting user's device location and this would be consistent regardless of what user has chosen. Perhaps we should differentiate that and maybe call it `GetDeviceRegion` `GetDesktopRegion` 

cc @bsclifton

Reply:
I thought `RegionList` and default region are not related with user's choice because they are provided data from guardian and user current timezone. It's like constant value. So, I thought they can be provided together.
Of course, we should store users choice separately in persistant storage like prefs. and it will be managed separately.

Reply:
but I don't have strong opinion about this. If you want to use separated `GetDeviceRegion` method, I could add it.

Reply:
Added `GetDeviceRegion()`.